### ISSUE TITLE
Coco: Format error?

### ISSUE THREAD

**Issue by RyanCV** [YonghaoHe/Light-And-Fast-Object-Detectors#46]
https://github.com/YonghaoHe/LFD-A-Light-and-Fast-Detector/blob/b1793c1a1571ff48ad15a61ab142e5df20d09ab3/lfd/data_pipeline/sampler/region_sampler.py#L52

The coco bbox format is: [x, y, w, h], then this line should be wrong.

**Comment by YonghaoHe** [comment]
@RyanCV Yes, some samplers do not support COCO format, you can write a new sampler on your own.
